docs/design_docs/index.md

```diff
@@ -0,0 +1,5 @@
+# Design Docs Index
+
+| Name                                        | Link                                                                                                    |
+| ------------------------------------------- | ------------------------------------------------------------------------------------------------------- |
+| Envoy Access Logging and Extension Services | [doc](https://docs.google.com/document/d/1CZo79XlSBgwr2dOrbR1YDYzPveuk6lr6cAIkl75rW3g/edit?usp=sharing) |
```

Comment:
Thanks ;-)

Reply:
Heh, no prob! At some point, someone can go and backfill our existing design docs; I just didn't have time for this initial PR